astropy/cosmology/_src/flrw/base.py

```diff
@@ -95,7 +96,14 @@ def quad(*args, **kwargs):
 
 
 @dataclass_decorator
-class FLRW(Cosmology, ScaleFactor, TemperatureCMB, _CriticalDensity, _BaryonComponent):
+class FLRW(
+    Cosmology,
+    ScaleFactor,
+    TemperatureCMB,
+    _CriticalDensity,
+    _BaryonComponent,
+    CurvatureComponent,
```

Comment:
Is there some reason these can't be alphabetically sorted?

Reply:
Done.